bin/cliain/src/main.rs

```diff
@@ -54,7 +55,14 @@ fn main() {
         command,
     } = Config::parse();
 
-    let seed = read_secret(seed, "Provide seed for the signer account:");
+    let seed = match command {
+        Command::Finalize {
```

Comment:
I guess RotateKeys and GetStorage also do not need the seed phrase, right? And also SeedToSS58 should take a seed as an argument to command not as a signer seed. Could we handle those in a similar manner?

Reply:
Done

Reply:
Could you also move this piece of code to a separate function. `main` is already packed...

Reply:
Done